schemas/2019-04-01/autogeneratedResources.json

```diff
@@ -261,6 +261,12 @@
         },
         {
           "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.DevSpaces.json#/resourceDefinitions/controllers"
+        },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.ContainerService.json#/resourceDefinitions/managedClusters"
+        },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.ContainerService.json#/resourceDefinitions/managedClusters_agentPools"
```

Comment:
Does managedClusters_agentPools also need to be added to the other root schemas?

Reply:
Probably not. I will remove this. It is an untracked child resource.